### ISSUE TITLE
v2.7 gpu window keeps closing then opening.

### ISSUE THREAD

**Issue by chabgood** [aaronsace/MultiPoolMiner#521]
The CPU window stays up, ut eery minute the GPU window closes then reopens.  I am sure this ha to due with the 60 sec default time, but the GPU window should stay up if there is no change in the current algorithm.

**Comment by fonyo** [comment]
Was this after benchmarking or during

**Comment by chabgood** [comment]
Yes.

**Comment by lee-aaron** [comment]
Could you attach supporting documentation? Also I'd like people to start using a format like [this](https://github.com/aaronsace/MultiPoolMiner/wiki/Bug-Reporting) in the future just so we can get to the root of bugs faster.

**Comment by fonyo** [comment]
If you require users to only submit extensive bug reports, you will scare most of them away as they won't even bother due to how much time it takes to key in the requirements thus reducing the number of bug reports as input for a better, more stable version of MPM. This is not a corporate application and I understand your good intentions with that wiki and probably somewhere in-between of having no rules and too many rules (requirements) would be great.

**Comment by fonyo** [comment]
@chabgood does this mean it is actually switching miners too frequently or it closes and reopens the same miner every minute? Post your start.bat as well please.

**Comment by aaronsace** [comment]
@lee-aaron It would go here: https://github.com/aaronsace/MultiPoolMiner/community

But I honestly think @fonyo is correct.

**Comment by aaronsace** [comment]
@chabgood Sounds like the miner keeps failing but not sure. Watchdog would detect this though. 

Check v2.7.0.1 and if you are still having problems then post your log: 
https://github.com/aaronsace/MultiPoolMiner/releases

**Comment by chabgood** [comment]
per the batch file:
powershell -version 5.0 -noexit -executionpolicy bypass -windowstyle maximized -command "&.\multipoolminer.ps1 -wallet 1HKBx8AX8vkrKzTcEQh7GQJ1qrNaoWDHjj -username aaronsace -workername multipoolminer -region us -currency btc,usd,eur -type nvidia,cpu -poolname nicehash -algorithm cryptonight,decrednicehash,ethash,equihash,lbry,neoscrypt,pascal,sianicehash,sib -donate 2"


It closes the same miner then reopens it.

**Comment by fonyo** [comment]
Thanks for that. Which miner is it, though? And what cards do you have?
Also, please head over to /logs and submit here the latest one.

**Comment by chabgood** [comment]
The algorithm is neoscrypt, I have 3 1070's.  The miner does not fail, it just stops.  I am attaching the logs.

[2017-11-21_16-45-49.txt](https://github.com/aaronsace/MultiPoolMiner/files/1496507/2017-11-21_16-45-49.txt)
[2017-11-21_18-01-23.txt](https://github.com/aaronsace/MultiPoolMiner/files/1496508/2017-11-21_18-01-23.txt)
[2017-11-21_18-12-15.txt](https://github.com/aaronsace/MultiPoolMiner/files/1496509/2017-11-21_18-12-15.txt)
[2017-11-21_18-32-56.txt](https://github.com/aaronsace/MultiPoolMiner/files/1496510/2017-11-21_18-32-56.txt)
[2017-11-21_19-40-19.txt](https://github.com/aaronsace/MultiPoolMiner/files/1496511/2017-11-21_19-40-19.txt)

**Comment by fonyo** [comment]
That seems to be bug just by looking at the log. There are two concurrent active miners list. And there is nothing in the idle group.

**Comment by chabgood** [comment]
I have the CPU and GPU both mining.

**Comment by fonyo** [comment]
You are right. My bad. I was actually looking at it on my phone and the tables were all quirky. But for some reason, it still says it is running equihash when it isn't. Would have to check the miner paths, maybe something got mixed up there. I am actually testing out a theory with nicehash and 1070s in a bit so I should be encountering the same issue. Will get back to you as soon as I'm finished. Please don't expect blazing speeds from me, I have only slept 1 hour since yesterday. In the meantime, it might be worth starting from scratch by redownloading the latest release.

**Comment by ndgouveia** [comment]
So, any update on this issue? I have the same issue, MPM just keeps closing and opening miners after 1 minute. I have AMD though, 8 RX580. Watchdog says it failed to connect to the miner even though the miners are working and submitting shares to MPH...
